spec/puppetserver/ca/action/prune_spec.rb

```diff
@@ -95,13 +95,21 @@
   end
 
   describe 'update' do
-    it 'bumps CRL version up by 1' do
+    it 'bumps CRL number up by 1' do
       ca_key = OpenSSL::PKey::RSA.new(512)
       ca_cert = create_cert(ca_key, "Bazzup")
       ca_crl = create_crl(ca_cert, ca_key)
 
       subject.update_pruned_CRL([ca_crl], ca_key)
-      expect(ca_crl.version).to eq(2)
+
+      # crlNumber is based on the amount of certs in revoked list
```

Comment:
I don't think that's true? It gets updated every time the CRL is updated, so _could_ happen to be the same as the number of revoked certs, but I think we allow batch revocations and also this pruning would mean that the CRL Number and number of revoked certs are not aligned.

Reply:
Ah, this is because of the util function `create_crl` that I used to build the crl.  That function uses the length of the revoked list to set the crl number and in the case of the spec test, I didn't pass in any array of revoked list so the length is 0.  Sorry if that comment is misleading.